src/CMakeLists.txt

```diff
@@ -24,26 +24,26 @@ include(GUDHI_compilation_flags)
 # Add your new module in the list, order is not important
```

Comment:
Same here, I don't think this is what we want to do. Can you make a separate cmakelists for fuzzing?